packages/airnode-node/src/handlers/call-api.ts

```diff
@@ -55,17 +54,19 @@ function buildOptions(
   };
 }
 
-function createPrintableValue(value: adapter.ValueType): string {
-  if (Array.isArray(value)) return JSON.stringify(value);
+export interface ApiCallOptions {
+  readonly forTestingGateway?: boolean;
+}
 
-  return BigNumber.isBigNumber(value) ? adapter.bigNumberToString(value) : value.toString();
+export interface CallApiPayload {
+  readonly config: Config;
+  readonly aggregatedApiCall: AggregatedApiCall;
+  readonly apiCallOptions: ApiCallOptions;
```

Comment:
Wouldn't be better to make this optional?

Reply:
By not making it optional I can enforce the AWS `callApi` handler to pass this value - so I'd say this is better.